### PR TITLE
Ewinner/designpatterns

### PR DESCRIPTION
**Pull request by ewinnerMSFT**
Add file generation based on guidance files in the playground extension.
Add handling of .md files for include-section into other md files

### CODE REVIEW

**Comment by mikekinsman** [comment]
Docs Build status updates of commit _[078c506](https://github.com/Azure/portaldocs/pull/210/commits/078c506b0c073b138628830e7ad2d7a24d766759)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-3c26c579)

<a id="3c26c579"></a>
###
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/Azure/portaldocs, working directory: W:\nrv0-s, target branch: dev, target branch commit: d8c21589e258b75075c1b2fa87ca17f29853c368, source branch: ewinner/designpatterns, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201904152150283326-210%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cVJTNaE96voQTYuHIRsTw64PR9vOsZVrlGP%2BOsNYuOo%3D&st=2019-04-15T21%3A45%3A41Z&se=2019-05-16T21%3A50%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
Docs Build status updates of commit _[282810c](https://github.com/Azure/portaldocs/pull/210/commits/282810c60ca13d89abb2d3f38402c9f421ab64f0)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-824da0d0)

<a id="824da0d0"></a>
###
 - **[Error]** Publish service&#39;s config file: .openpublishing.publish.config.json is not found. Exception: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C3d86255f-f488-188e-02a3-9e92aa67830d%5CPullRequest%5C201904152154187757-210%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WYE0M0GqD%2B9yD%2FCav3oGz%2BazGs%2FMDf2wn2iaipqZBok%3D&st=2019-04-15T21%3A49%3A31Z&se=2019-05-16T21%3A54%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: portal-sdk/generated/index-portalfx-extension-monitor.md [hunk 1]
```diff
@@ -70,44 +70,16 @@
```
Comment (ShrishaKumar) [inline portal-sdk/generated/index-portalfx-extension-monitor.md]: Is this an intended change or a consequence of npm run docs?

File: portal-sdk/generated/design-patterns-controls-button.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+﻿# Button
+
+ 
+<a name="basics"></a>
+### Basics
+Buttons are best used to enable a user to commit a change or complete steps in a task. They are typically found inside forms, dialogs, panels or pages. An example of their usage is confirming the deletion of a resource in a confirmation dialog.
+
+
+<!-- EWINNER TODO get an IMAGE to embed here -->
```
[the remark this answers is not in the document]
Comment (ewinnerMSFT) [reply portal-sdk/generated/design-patterns-controls-button.md]: Good to know - I'll clean it out at some point

**Pull request merged**